src/myst/vm.cr

```diff
@@ -6,12 +6,12 @@ module Myst
     # Instead `for_file` and `for_content` should be used
     # Many of these parameters would be rarely used
     # All having default values, i don't really think the amount of parameters is a problem
-    def initialize(source : IO = IO::Memory.new, source_name : String = "eval_input", with_stdlib? : Bool = true,
-                   use_stdios? : Bool = false, product : Bool = true)            
+    def initialize(source : IO = IO::Memory.new, *, source_name : String = "eval_input", with_stdlib? : Bool = true,
+                   use_stdios? : Bool = false, product? : Bool = true)            
```

Comment:
what's the `*` parameter used for? Just to indicate that the remaining parameters must be passed by name?

Reply:
Yes